src/sagas.ts

```diff
@@ -0,0 +1,24 @@
+import { call, put, takeLatest } from 'redux-saga/effects';
+import fetchCounters from './api';
+import {
```

Comment:
This looks good, the only change I'd request is to put sagas into a /saga folder.